hedera-node/docker/Dockerfile

```
@@ -1,42 +1,48 @@
-## Installs OpenJDK17 and openssl (used by Swirlds Platform to
-## generate node keys for e.g. signing states), then copies
+## Starts from the recent release base image then copies
 ## required libraries and startup assets for a node with:
-##  * Configuration from /opt/hedera/services/config-mount; and,
-##  * Logs at /opt/hedera/services/output; and,
-##  * Saved states under /opt/hedera/services/output
-FROM ubuntu:20.04 AS base-runtime
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y dos2unix openssl libsodium23 bind9-host
-
-# JDK
-RUN apt-get install -y software-properties-common && \
-    add-apt-repository -y ppa:openjdk-r/ppa && \
-    apt-get install -y openjdk-17-jdk
-
-# Services runtime
-RUN mkdir -p /opt/hedera/services/data/lib
-RUN mkdir /opt/hedera/services/data/apps
-RUN mkdir /opt/hedera/services/data/config
-RUN mkdir /opt/hedera/services/data/saved
-RUN mkdir /opt/hedera/services/data/onboard
-RUN mkdir /opt/hedera/services/output
-RUN mkdir /opt/hedera/services/config-mount
-
-## Finishes by copying the Services JAR to the base runtime
-FROM base-runtime AS final-image
-
-WORKDIR /opt/hedera/services
-
-COPY start-services.sh /opt/hedera/services/start-services.sh
-
-COPY .env /opt/hedera/services
+##  * Configuration from /opt/hgcapp/services-hedera/HapiApp2.0/config-mount; and,
+##  * Logs at /opt/hgcapp/services-hedera/HapiApp2.0/output; and,
+##  * Saved states under /opt/hgcapp/services-hedera/HapiApp2.0/output
+ARG IMAGE_PREFIX="gcr.io/hedera-registry/"
+ARG IMAGE_TAG="0.43.0-alpha.6"
+
+FROM ${IMAGE_PREFIX}network-node-base:${IMAGE_TAG} AS final-image
```

the `AS` clause can be dropped since this is no longer a multi-stage build

```suggestion
FROM ${IMAGE_PREFIX}network-node-base:${IMAGE_TAG}
```